controller/src/main/java/io/pravega/controller/store/stream/ZKScope.java

```diff
@@ -298,13 +298,13 @@ protected void declareVersions() {
             private void read00(RevisionDataInput revisionDataInput, Token.TokenBuilder builder) throws IOException {
                 builder.msb(revisionDataInput.readCompactInt())
                        .middle(revisionDataInput.readCompactInt())
-                       .lsb(revisionDataInput.readCompactInt());
+                       .lsb(revisionDataInput.readInt());
```

Comment:
I don't understand this change.

Reply:
Compact int takes only non negative values. token.EMPTY has long.min value as lab.. which is cannot be serialised using compact int. Hence changed it to int

Reply:
Can't we use a different encoding for empty rather than replacing compact int? Using compact encoding as much as we can is definitely preferable.

Reply:
I don't understand, you 👍 it, but I don't see a change.

Reply:
Sure. I can encode msb with a value greater than that number of top level znodes (which is 100). 
following is the tree structure:
100 -> 10k -> 10k
So encoding MSB with anything greater than 100 can be one "hacky" way. 
I didnt quite like that. 
These continuation tokens are only sent over the wire and 2 additional bytes for non-critical query api call (`listStreams`) does not seem too expensive.